src/integrationTest/java/uk/gov/hmcts/reform/iacasedocumentsapi/component/GenerateDecisionAndReasonsTestWiremock.java

```diff
@@ -63,6 +65,8 @@ public void generates_decision_and_reasons() {
                     .with(APPELLANT_REPRESENTATIVE, "ted")
                     .with(RESPONDENT_REPRESENTATIVE, "bill")
                     .with(LIST_CASE_HEARING_CENTRE, "taylorHouse")
+                    .with(APPEAL_TYPE, AppealType.HU.getValue())
+                    .with(REMISSION_TYPE, RemissionType.NO_REMISSION)
```

Comment:
You don't need this anymore, since you removed the "Remission type not found" check.

https://github.com/hmcts/ia-case-documents-api/compare/1fccc4285b5ebbb784130e6c0e90747952a0237e..6dcc673cb3873b744cf750fae71c852277baa712